cocotb/decorators.py

```diff
@@ -422,6 +423,8 @@ class test(with_metaclass(_decorator_helper, coroutine)):
     Args:
         timeout (int, optional):
             value representing simulation timeout (not implemented).
+        timeout_units (str, optional):
+            units of the timeout value.  See Timer for valid values.
```

Comment:
I know that that `Timer` uses `units` as name, but having it plural there always sounded like a bug to me.

Reply:
I don't disagree, I did it for consistency with Timer and Clock.  I could do a pull request that would fix that while maintaining backwards compatibility and warn of deprecated argument.  Is that something you want?

Reply:
Plural sounds correct to me. Can you point to something indicating it's not, @imphil?

Reply:
`timeout` is singular (it's 1 value), and it this value is associated with 1 unit.